packages/dcos-log/extra/dcos-log-master.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=DC/OS Log service master: DC/OS logging service
+Description=DC/OS Log Master: exposes component, container, and task logs
```

Comment:
Or is this more accurate?
- master dcos-log expose component logs
- agent dcos-log exposes container/task logs
- admin router exposes both to the external user via proxy